tests/testthat/test-fimsfit.R

```diff
@@ -9,10 +9,12 @@
 # FIMSFit ----
 ## Setup ----
 # Load the test data from an RDS file containing the fitted model estimates
+if (!file.exists(test_path("fixtures", "fit_age_length_comp.RDS"))) {
```

Comment:
All of these if statements are pretty brilliant so the tests don't take forever because it is just repeating the same process. Great work 🥇